packages/neuron-ui/src/components/ApproveMultisigTxDialog/index.tsx

```diff
@@ -16,7 +17,7 @@ import styles from './approveMultisigTx.module.scss'
 const Cell = React.memo(
   ({ cell, isMainnet }: { cell: State.DetailedInput | State.DetailedOutput; isMainnet: boolean }) => {
     const address = useMemo(
-      () => (cell.lock ? ckbCore.utils.scriptToAddress(cell.lock, isMainnet) : ''),
+      () => (cell.lock ? encodeToAddress(cell.lock, { config: isMainnet ? predefined.LINA : predefined.AGGRON4 }) : ''),
```

Comment:
There is already a script/address conversion API implemented in Neuron, maybe we can use it directly. https://github.com/nervosnetwork/neuron/blob/4d1b3bbb3540a833c974b770f6758daf0efbb291/packages/neuron-wallet/src/utils/scriptAndAddress.ts#L5

Reply:
Good to know it. 
But `scriptAndAddress.ts` is in `neuron-wallet` so we can't use it in this PR, 
because the changes in this PR are in `neuron-ui`.

Reply:
I noticed that there are several instances of `{ config: isMainnet ? predefined.LINA : predefined.AGGRON4 })` in this PR. 

Copy-pasting code with logic over three times is not recommended as it can increase complexity. Moreover, `predefined.LINA` and `predefined.AGGRON4` are outdated network names since version 0.22, which were changed to `MAINNET` and `TESTNET`. It is better to refactor the code to avoid increasing complexity

Reply:
We can replace `predefined.LINA` `predefined.AGGRON4` with `MAINNET` and `TESTNET` 
when we upgrade to lumos 0.22, so let's put that aside for now.

For `{ config: isMainnet ? predefined.LINA : predefined.AGGRON4 })`   
Let's see the problem from another angle, I can write wrapper functions like
```ts
import { encodeToAddress, generateAddress as _generateAddress } from "@ckb-lumos/helpers";

function generateAddress(script: Script, isMainnet: boolean) {
   return _generateAddress(script, { config: isMainnet ? predefined.LINA : predefined.AGGRON4 }))
}

function scriptToAddress(script: Script, isMainnet: boolean) {
   return encodeToAddress(script, { config: isMainnet ? predefined.LINA : predefined.AGGRON4 }))
}

... Wrap other lumos methods
```
It becomes
`Project code` calling `Wrappers of Lumos methods` calling `Lumos methods`
This means Lumos doesn't solve my problem directly, 
I have to wrap it inside another function to to make things work.

But what we/developers want is
`Project code` calling `Lumos methods`
This means Lumos solves my problem directly.

Reply:
It is common in development to use 3rdparty libraries within an application. Neuron has an address-related method that involves only two networks - the mainnet and non-mainnet. This makes it possible for Neuron to only require a boolean to control the address conversion instead of a complete `Config`.

Therefore, it is reasonable to have a wrapper for the address conversion in Neuron that uses the following syntax:

```ts
declare function somethingAddress(value: Something, isMainnet: boolean): Address
```

Reply:
Added `neuron-ui/src/utils/scriptAndAddress.ts` and calling `scriptToAddress` & `addressToScript` now.

@homura Can you trigger the CI workflows ?

```ts
import { predefined } from '@ckb-lumos/config-manager'
import { encodeToAddress, parseAddress, generateAddress } from '@ckb-lumos/helpers'

const { LINA: MAINNET, AGGRON4: TESTNET } = predefined

const CONFIGS = {
  [MAINNET.PREFIX]: MAINNET,
  [TESTNET.PREFIX]: TESTNET,
}

export const scriptToAddress = (
  script: CKBComponents.Script,
  { isMainnet = true, deprecated = false }: { isMainnet?: boolean; deprecated?: boolean }
) => {
  const config = { config: isMainnet ? MAINNET : TESTNET }
  return deprecated ? generateAddress(script, config) : encodeToAddress(script, config)
}

export const addressToScript = (address: string, { isMainnet = true }: { isMainnet?: boolean } = {}) => {
  const prefix = address.slice(0, 3)
  const config = CONFIGS[prefix] ?? (isMainnet ? MAINNET : TESTNET)
  return parseAddress(address, { config })
}

```